src/clj/rems/db/applications.clj

```diff
@@ -410,6 +410,26 @@
       (:filename (db/get-attachment query-params))
       (:value (db/get-field-value query-params)))))
 
+(defn- process-item-options [options]
+  (->> options
+       (map (fn [{:keys [key langcode label displayorder]}]
+              {:key key
+               :label {(keyword langcode) label}
+               :displayorder displayorder}))
```

Comment:
Isn't displayorder unnecessary at least outside SQL layer if you have a sequence always?

Reply:
Yes. The displayorder is dropped by the select-keys operation a few rows below this.

Reply:
Due to the group-by operation the displayorder is still needed in this function.